format-common/src/main/java/io/cdap/plugin/format/input/AbstractEmptyInputFormat.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2018-2019 Cask Data, Inc.
```

Comment:
it is showing it as new file. make it 2024 here and 2018-19 in emptyinputformat

Reply:
What about `EmptyInputFormat` change it back to 18-19 ?

Reply:
yes

Reply:
Fixed All !